ast/src/main/java/com/graphicsfuzz/common/tool/PrettyPrinterVisitor.java

```diff
@@ -607,6 +674,32 @@ public void visitExprCaseLabel(ExprCaseLabel exprCaseLabel) {
 
   @Override
   public void visitInterfaceBlock(InterfaceBlock interfaceBlock) {
+
```

Comment:
A comment analogous to the comment for the variables case ("// If uniform variables are being declared and we have ...") would be appropriate here - but talking about a uniform block instead of uniform variables.